pkg/config/config_test.go

```diff
@@ -132,6 +133,7 @@ func Test_ExtractConfigData(t *testing.T) {
  segreaderNode: true
  metareaderNode: true
  segFlushIntervalSecs: 1200
+ segWipFlushMaxIntervalSecs: 300
```

Comment:
if we are only allowing a value upto 60 then setting this to 300 does not make sense